ocp_resources/node_network_configuration_policy.py

```diff
@@ -326,6 +331,25 @@ def clean_up(self):
 
         super().clean_up()
 
+    def append(self, nncp, spec_to_append):
+        """
+        It's the responsibility of the caller to verify the desired configuration they send.
+        This function won't check that appending an nncp won't break config.
```

Comment:
please use proper docstring format and content

Reply:
Wdym? Am I missing something except for example?

Reply:
see https://peps.python.org/pep-0257/